test/tools/router_check/json/tool_config_schemas.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace Json {
+
+class ToolSchema {
+
+public:
+  /**
+   * Obtain the router check json schema
+   * @return const std::string& of the schema string
+   */
+  static const std::string& routerCheckSchema();
```

Comment:
Can you create a static const var like https://github.com/lyft/envoy/blob/master/source/common/json/config_schemas.h#L8?